### PR TITLE
Move all the default value set into the fetch method

### LINKED ISSUE #450: Move the default value setup into the common fetch method
/kind feature

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

### PR DESCRIPTION
**Pull request by DanielXLee**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #450 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/controller/operandconfig/operandconfig_controller.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ import (
 	"context"
 	"time"
 
+	operatorv1alpha1 "github.com/IBM/operand-deployment-lifecycle-manager/pkg/apis/operator/v1alpha1"
+	fetch "github.com/IBM/operand-deployment-lifecycle-manager/pkg/controller/common"
```
Comment (horis233) [inline pkg/controller/operandconfig/operandconfig_controller.go:24]: @DanielXLee Should we divide the internal import from the third party library?

File: pkg/controller/operandregistry/operandregistry_controller.go [hunk 2]
```diff
@@ -21,6 +21,8 @@ import (
 	"reflect"
 	"time"
 
+	operatorv1alpha1 "github.com/IBM/operand-deployment-lifecycle-manager/pkg/apis/operator/v1alpha1"
```
Comment (horis233) [inline pkg/controller/operandregistry/operandregistry_controller.go:24]: @DanielXLee Should we divide the internal import from the third party library?

File: pkg/controller/operandconfig/operandconfig_controller.go (hunk 1 above)
Comment (DanielXLee) [reply pkg/controller/operandconfig/operandconfig_controller.go:24]: okay, divide them is better.

**Pull request merged**